test/e2e/set/scyllacluster/scyllacluster_tls.go

```diff
@@ -292,7 +310,6 @@ var _ = g.Describe("ScyllaCluster", func() {
 				o.Expect(initialSecret.Data).To(o.HaveKey("tls.key"))
 				o.Expect(initialSecret.Data["tls.crt"]).NotTo(o.BeEmpty())
 				o.Expect(initialSecret.Data["tls.key"]).NotTo(o.BeEmpty())
-				o.Expect(initialSecret.Annotations).To(o.HaveKeyWithValue("certificates.internal.scylla-operator.scylladb.com/refresh-reason", "needs new cert"))
```

Comment:
as the cert was generated in phases, was this no longer deterministic?

Reply:
Before this change PodIP wasn't added to certificate at all. PodIP is available after Pod initializes so before SC rolls out, cert created and then updated so reason no longer matches.